skatepark/inc/patterns/text-list-with-button.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Text list with Button.
+ *
+ * @package Skatepark
+ * @since 1.0.0
+ * 
+ */
+
+return array(
+	'title'      => __( 'Text List with Button', 'skatepark' ),
+	'categories' => array( 'skatepark' ),
+	'content'    => '<!-- wp:group -->
+		<div class="wp-block-group"><!-- wp:heading {"level":4,"className":"is-style-skatepark-heading-border"} -->
```

Comment:
Should this be alignwide in the pattern?

Reply:
To me, it looks like a wide align container with the content aligned to the left with a max-width of 888px. Without wide align the content can not be wider than 684px, so it won't look like the design:

<img width="799" alt="Screenshot 2021-08-12 at 09 26 55" src="https://user-images.githubusercontent.com/3593343/129155665-d4b57ed4-5b56-4cdf-92f7-86dc71922a3f.png">